sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -282,6 +286,13 @@ def plot_partial_dependence(
 
         .. versionadded:: 0.24
 
+    centered : bool, default=False
+        Only has an effect when `kind='both'` or `kind='individual'`. When `True`,
+        the ICE and PD lines are centered around the origin of the x-axis. By
+        default, no centering is done.
```

Comment:
It's a bit weird that average pdp can only be centered if it comes with the individual ones. We could always center even for kind="average"

Reply:
After using it for the test, I agree that this is a bit weird. We could centre the average PD as well.